ironfish/src/rpc/server.ts

```diff
@@ -73,4 +76,21 @@ export class RpcServer {
       this._startPromise = promise
     }
   }
+
+  /** Authenticate the RPC request */
+  authenticate(requestAuthToken: string | undefined): boolean {
+    if (requestAuthToken) {
```

Comment:
Should this be `if (!requestAuthToken)`? It looks like `authenticate` will always return false